src/quicktype-core/support/Acronyms.ts

```diff
@@ -1095,3 +1098,34 @@ export const acronyms: string[] = [
     "zope",
     "zpl"
 ];
+
+export enum AcronymStyleOptions {
+    Pascal,
+    Camel,
+    Capitalize,
+    OriginalWord
+}
+
+export const acronymOption = new EnumOption(
```

Comment:
Could you make this into a function that takes the default as an argument?  Not all languages will have `pascal` as their default.

Reply:
Sure, I'll do the change now